hands-on-exercise/4-run-in-prod/2-migration.md

```diff
@@ -1731,8 +1731,8 @@ From another shell, create a few un-played games with:
 <CodeGroupItem title="Local" active>
 
 ```sh
-$ export alice=$(./release/v1/checkersd keys show alice -a)
-$ export bob=$(./release/v1/checkersd keys show bob -a)
+$ export alice=$(./release/v1/checkersd keys show alice -a --keyring-backend test)
+$ export bob=$(./release/v1/checkersd keys show bob -a --keyring-backend test)
```

Comment:
I would rather there be another paragraph, or troubleshooting box, saying if you have unrecognised keys, you may have to always add the keyring.

Reply:
@xavierlepretre can you resolve this please?